client/src/modules/journal/modals/search.modal.html

```diff
@@ -30,14 +30,16 @@
           </div>
         </div>
 
-        <bh-account-select
-          account-id="ModalCtrl.options.account_id"
-          name="account_id"
-          on-select-callback="ModalCtrl.onSelectAccount(account)"
-          exclude-title-accounts="true"
-          required="0">
-          <bh-clear on-clear="ModalCtrl.clear('account_id')"></bh-clear>
-        </bh-account-select>
+        <div ng-if="!ModalCtrl.hasDefaultAccount">
```

Comment:
A comment explaining when this switch is used would be useful for developers working on either the Journal or the Account Statement module.